pkg/securitycontext/util.go

```diff
@@ -61,12 +62,13 @@ func DetermineEffectiveSecurityContext(pod *v1.Pod, container *v1.Container) *v1
 		return containerSc
 	}
 
-	if containerSc.SELinuxOptions != nil {
+	// To preserve backwards compatibility
+	if runtime.GOOS != "windows" && effectiveSc != nil && containerSc != nil && containerSc.SELinuxOptions != nil {
```

Comment:
The `effectiveSc != nil && containerSc != nil` checks seem to be taken care of by the checks [above](https://github.com/kubernetes/kubernetes/blob/3365ed5625797cfaafbdab368e901e9f4739329a/pkg/securitycontext/util.go#L57-L62) but I think this is safer code.